### PR TITLE
build/tools/amebad: Add Jlink Debug Function

### PR DESCRIPTION
**Pull request by AdamZhang0124**
Please read the 'README.md' for the use of the Jlink debug function.

### CODE REVIEW

File: build/tools/amebad/gnu_utility/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# AmebaD Jlink Debug Function
+
+### Requirements
```
Comment (jeongchanKim) [inline build/tools/amebad/gnu_utility/README.md:3]: If specific version is not needed, I think we can remove this.

File: build/tools/amebad/gnu_utility/AP2_KM4.JLinkScript [hunk 2]
```diff
@@ -0,0 +1,9 @@
+void InitTarget(void) {
+	// Set AP info for all APs in the device
+	CORESIGHT_AddAP(0, CORESIGHT_APB_AP);
+	CORESIGHT_AddAP(1, CORESIGHT_AHB_AP);
+	CORESIGHT_AddAP(2, CORESIGHT_AHB_AP);
+	CORESIGHT_IndexAHBAPToUse = 2;
+	JTAG_WriteClocks(1);	// Make sure that JTAG buffers are empty
+JTAG_Speed = 1000;
```
Comment (jeongchanKim) [inline build/tools/amebad/gnu_utility/AP2_KM4.JLinkScript:8]: Please make alignment.

Comment (AdamZhang0124) [reply build/tools/amebad/gnu_utility/AP2_KM4.JLinkScript:8]: Sure, thanks!

File: build/tools/amebad/gnu_utility/README.md (hunk 1 above)
Comment (AdamZhang0124) [reply build/tools/amebad/gnu_utility/README.md:3]: Ok, I will remove it, thanks!

**Pull request merged**